src/dataflow/src/source/kinesis.rs

```diff
@@ -7,26 +7,30 @@
 // the Business Source License, use of this software will be governed
 // by the Apache License, Version 2.0.
 
-use std::collections::{HashMap, HashSet, VecDeque};
-use std::time::{Duration, SystemTime, UNIX_EPOCH};
-
+use crate::server::{
```

Comment:
The import order will need to be changed here so it matches:
- std imports
- other 3rd party imports
- non-crate, materialize imports
- crate imports